inlineplz/decorators.py

```diff
@@ -54,5 +55,6 @@ def linter(
             "patterns": patterns,
             "url": url,
             "always_install": always_install,
+            "autofix": autofix
```

Comment:
I might be missing something, but this makes it look like a linter is globally configured as autofix or not? So the built-in linters might need to be disabled and duplicated just to turn off autofix for a repo?

Reply:
hmm my intention is that you have to explicitly opt-in to autofixing by setting `autofix: true` in your `.inlineplz.yml` - otherwise the autofixers will not run.

Reply:
I see that. It looks to me like a linter is defined to either be an autofixer or not, though? so if I opt in to autofixers, I get all of the autofixers doing their autofixing. There's no way for me to have one autofixer fix, and another just report without duplicating the other to redefine it to not be an autofixer. (For that matter, there's also no way to opt-in to python autofixing without also opting-in to golang autofixing.)

Reply:
ah, i see. yes, my current design is that either it's an autofixer or it's not. i figure that most automatic code formatters are not designed to be run as static analysis tools that can report errors line-by-line.

Reply:
also at least as an initial implementation i figure people either want autofixing for everything or for nothing.